package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@tforster/webproducer",
-  "version": "0.7.1",
+  "version": "0.9.0",
```

Comment:
Is there a `v0.8.0` to merge to `main` or is that being skipped?

Reply:
Skipped. It has been difficult to maintain stable versions with WebProducer being developed for different consuming applications. This will likely be the last time I update main. I am going to switch to a model that doesn't use the main branch anymore (or put another way, getting rid of develop and using just main. Either way, reducing the number of top level branches from two to one)